components/scream/src/share/io/scorpio_output.cpp

```diff
@@ -85,11 +85,10 @@ void AtmosphereOutput::init()
     rhist_in.init();
     for (auto name : m_fields)
     {
-      auto l_view = m_view_local.at(name);
-      rhist_in.pull_input(name,l_view);
+      auto l_view = rhist_in.pull_input(name);
+      Kokkos::deep_copy(m_view_local.at(name),l_view);
```

Comment:
Why the deep copy? Why not simply `m_view_local.at(name) = l_view'`?

Reply:
I guess not, thanks :-)